src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php

```diff
@@ -99,7 +99,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if (!$socket = stream_socket_server($host, $errno, $errstr)) {
-            throw new RuntimeException(sprintf('Server start failed on "%s": %s %s.', $host, $errstr, $errno));
+            throw new RuntimeException(sprintf('Server start failed on "%s": '.$errstr.' '.$errno, $host));
```

Comment:
The concatenation should be outside the `sprintf`. Otherwise, you need to escape `%` in `$errstr`